README.md

```diff
@@ -39,16 +41,82 @@ query = QueryBase(
 print("Results available at", query.url())
 
 dune = DuneClient.from_env()
-results = dune.refresh(query)
+results = dune.run_query(query)
+
+# or as CSV
+# results_csv = dune.run_query_csv(query)
+
+# or as Pandas Dataframe
+# results_df = dune.run_query_dataframe(query)
 ```
 
-For a more elaborate example,
-visit [dune-alerts](https://github.com/cowprotocol/dune-alerts)
+## Further Examples
+
+### Get Latest Results
+Save execution credits with `get_latest_results`! Specify `max_age_hours` when using 
+latest result will trigger refresh if data is considered too old.
```

Comment:
"Use `get_latest_results` to get the most recent query results without using execution credits. You can specify a `max_age_hours` to re-run the query if the data is too outdated."